### PR TITLE
Change effect states in the Audio Renderer

### PR DESCRIPTION
**Pull request by Dudejoe870**
This is probably part of the issue why SMOs sub-areas don't work, this needs to be tested.  New Game in SMO still doesn't work, could be different with sub-areas though.

Has been tested: doesn't seem like sub-areas work, however, this could still be part of the problem.

### CODE REVIEW

File: Ryujinx.HLE/HOS/Services/Aud/AudioRenderer/IAudioRenderer.cs [hunk 1]
```diff
@@ -75,6 +77,8 @@ class IAudioRenderer : IpcService, IDisposable
 
             _voices = CreateArray<VoiceContext>(Params.VoiceCount);
```
Comment (emmauss) [inline Ryujinx.HLE/HOS/Services/Aud/AudioRenderer/IAudioRenderer.cs]: you could remove the new line, and align this two

**Comment by Dudejoe870** [comment]
Addressed Feedback

**Comment by BaronKiko** [comment]
Can this be merged with a rebase or should it be closed?

**Comment by Dudejoe870** [comment]
It should probably be closed, even though it has no merge conflicts, I'm not really that happy with the code and it is a bit of a hack to be honest.

**Pull request closed without merging**